### PR TITLE
Co-locate tests

### PR DESCRIPTION
**Pull request by dignite**
# Test structure

As a developer
In order to promote testing and to ease navigation
I want the tests close to the code

Related: https://github.com/mob-timer/mob-timer/issues/5

## Integration tests

In this folder (`/test`) you should only put integration tests that care equally about several modules, or tests that don't relate to any single module.

Example: _The tests in `node-version.test.js` make sure that the node.js versions are consistent in tests, in the app and in the CI._

## Unit tests

For unit tests, co-locate the test next to the module under test. Tests that are integration tests but have a clear main module under test could also be co-located.

Example: _The tests in `timer.test.js` check that the module `timer.js` behaves correctly, and both files are located in the same folder (somewhere inside the `src/` folder)._

### CODE REVIEW

File: src/state/test-timer.js [hunk 1]
```diff
@@ -1,21 +1,21 @@
-class TestTimer {
```
Comment (dignite) [inline src/state/test-timer.js]: Unclear why this file diffs 🤔

File: test/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Test structure
+
+## Integration tests
+
+In this folder (`/test`) you should only put integration tests that care equally about several modules, or tests that don't relate to any single module.
```
Comment (niklas-r) [inline test/README.md:5]: This folder could also house test helpers if we see a need for any.

**Review by niklas-r — APPROVED** [review]
Good stuff. Keeping tests close to the implementation is a good thing 👍

File: test/README.md (hunk 2 above)
Comment (dignite) [reply test/README.md:5]: True, we already have the test-timer.js which I located with the tests. Should that one be here?

**Pull request merged**